src/main/play-doc/developer/AkkaAndPlay.md

```diff
@@ -119,6 +119,25 @@ In the above, no declaration of `services` is required as akka remoting is an in
 
+#### Downing Strategies
+
+[The Akka documentation does not recommend auto-downing](http://doc.akka.io/docs/akka/2.4.6/scala/cluster-usage.html#Automatic_vs__Manual_Downing) for use within your Akka cluster based application/service. Instead it recommends that you manually shutdown a non-responsive service or provide a more robust automated strategy, such as using the Reactive Platform based Split Brain Resolver (SBR) package.
+
+As a subscriber to the Lightbend Reactive Platform you can use SBR as an automated downing mechanism. In this case note that a) the downing strategy may conflict with ConductR's (ConductR's is "keep majority"); and b) that you must handle the `MemberRemoved` event if it is received for the current member - SBR will not shutdown your application/service automatically. 
+
+In the case of contending strategies, this may not be a problem depending on your application/service's availability requirements. Suppose that your application/service chooses "keep oldest" as a strategy, and that the oldest member is in the minority portion of ConductR's cluster. If ConductR's SBR's "keep majority" decides to shutdown your oldest member, it will re-schedule it for execution again, but in the majority side of the cluster. In the case of "keep-oldest" resulting in reducing the number of running instances to one, your system may suffer an outage in terms of the length of time it takes ConductR to reschedule it for execution on the majority side of the cluster (seconds). This problem can be further mitigated by using an SBR timeout for "keep oldest" that is greater than ConductR's. 
+
+A reasonable start point when considering an SBR based strategy to use in conjunction with ConductR is to use the following configuration:
+
+```
+akka.cluster.split-brain-resolver.active-strategy = keep-majority
+akka.cluster.down-removal-margin = 30 seconds
+```
+
+> Always consult the Akka SBR documentation for details on the above parameters and to determine the best strategy for your particular application/service.
+
+With any downing strategy, you should consider the availability requirements of your application/service. If the application/service requires high availability then we suggest running at least 3 instances of it, and always an odd number in order to facilitate downing strategy decisions that rely on quorums (such as "keep majority").
```

Comment:
keep-majority also works for even numbers, but I guess the user must not no every little detail here